src/index.html

```diff
@@ -15,8 +15,49 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="0"
```

Comment:
The 'crossorigin' attribute should be set to 'anonymous' or 'use-credentials'. '0' is not a valid value for this attribute.